packages/remix/src/utils/instrumentServer.ts

```diff
@@ -318,7 +387,7 @@ function makeWrappedCreateRequestHandler(
 
     const requestHandler = origCreateRequestHandler.call(this, { ...build, routes, entry: wrappedEntry }, mode);
 
-    return wrapRequestHandler(requestHandler);
+    return wrapRequestHandler(requestHandler, build);
```

Comment:
This is working, but just in case anything changes internally in `@remix/server-runtime` (like comparing the `build` or `route` objects or anything), could we use the same `build` object we provide to `origCreateRequestHandler` here?

Reply:
Good idea, will do